nano/node/active_transactions.cpp

```diff
@@ -210,8 +215,8 @@ int64_t nano::active_transactions::vacancy (nano::election_behavior behavior) co
 		case nano::election_behavior::normal:
 			return limit () - static_cast<int64_t> (roots.size ());
 		case nano::election_behavior::hinted:
+		case nano::election_behavior::optimistic:
 			return limit (nano::election_behavior::hinted) - count_by_behavior[nano::election_behavior::hinted];
```

Comment:
@clemahieu Merge introduced a small bug here, it should pass in 'behavior' parameter instead of fixed enum value.